wiki/en/Installation-for-Macintosh.md

```diff
@@ -13,22 +13,9 @@ Make sure you've already read the [Getting Started](Getting-Started) page.
 
 Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.
 
-We provide three downloads for macOS. Please download the appropriate one:
-
-**For macOS running on Intel:**
-
-[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}
-
-For macOS Mojave (10.14 or lower) please [download the legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})
-
-**For macOS running on Apple Silicon:**
-
-[Download Jamulus (Apple Silicon)]({{ site.download_root_link }}{{ site.download_file_names.mac-arm }}){: .button}
-
+1. [Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\
+ **macOS Mojave (10.14) or lower:** [Download legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})\\
```

Comment:
Is "universal build" standard vocabulary for Mac users? What happens if we just give a minimum OS version?

Reply:
Yes, I think so. ![image](https://user-images.githubusercontent.com/20726856/188209290-4fca9362-1142-4801-8201-3c9d94b72d42.jpeg)

Reply:
OK well in that case it looks good to me.